controlpanel/api/models/app.py

```diff
@@ -305,6 +306,9 @@ def get_auth0_group_list(self):
     def auth_settings(self):
         return (self.app_conf or {}).get(self.KEY_WORD_FOR_AUTH_SETTINGS, {})
 
+    def get_absolute_url(self):
+        return reverse("manage-app", kwargs={"pk": self.id})
```

Comment:
```suggestion
        return reverse("manage-app", kwargs={"pk": self.pk})
```
Django sets a property to return the primary key which is useful in methods like this https://docs.djangoproject.com/en/5.0/ref/models/instances/#the-pk-property